spec/lib/doi_search_spec.rb

```diff
@@ -1,16 +1,21 @@
 
 describe DoiSearch do
-  let(:publication) { create :publication }
-  let(:author) { create :author }
+  let(:doi_value) { '10.1016/j.mcn.2012.03.007' }
+  let(:doi_identifier) do
```

Comment:
This is the same as the above `:doi_identifier`, so you can back out one to the common context and keep it in common.  

It might seem like there is an ordering issue (because `doi_identifier` depends on `doi_value`), but `let`  statements are not executed in the order declared (rather, in the order invoked in the example), so it would work fine.

Reply:
Huh?  It's in the global context for this spec file already.  Maybe your referring to the spec above, but these are different spec files and the `doi_value` is different between the two files (the last digit is different).

Reply:
OK, that makes sense.